packages/ibm-products/src/components/TagSet/TagSetOverflow.js

```diff
@@ -94,7 +94,13 @@ export const TagSetOverflow = React.forwardRef(
           </Tag>
           <PopoverContent>
             <div ref={overflowTagContent} className={`${blockClass}__content`}>
-              <ul className={`${blockClass}__tag-list`}>
+              <ul
+                className={`${blockClass}__tag-list ${
```

Comment:
We use the `classnames` package for handling conditional classnames like this. You can use it like this:
```jsx
import cx from 'classnames';
....
....
<ul
  className={cx(`${blockClass}__tag-list, {
    [`${blockClass}__tag-list--alt`]: overflowTags.length > allTagsModalSearchThreshold
  })}
>
```

Reply:
Thanks for the review!, i've addressed the comments.